contracts/README.md

```diff
@@ -0,0 +1,180 @@
+# Devnet contracts for development
+
```

Comment:
I would suggest also to rename the folder from /contracts to maybe /l1-l2-e2e or something better

Reply:
Now [contracts folder](https://github.com/glihm/starknet-devnet-rs/tree/7dd5d6e0fe8f14aca40ebd562cc3c5d5e16ce873/contracts) contains sub-folders with important files, and `l1-l2-messaging` sub-folder for messaging related contracts. Is that ok?

Reply:
fine by me! :)